src/Cardano/AddressDerivation/Byron.hs

```diff
@@ -0,0 +1,252 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE InstanceSigs #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RoleAnnotations #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
+-- License: Apache-2.0
+--
+-- Implementation of address derivation for the random scheme, as
+-- implemented by the legacy Cardano wallets.
+--
+-- For full documentation of the key derivation schemes,
+-- see the "Cardano.Crypto.Wallet" module, and the implementation in
+-- <https://github.com/input-output-hk/cardano-crypto/blob/4590efa638397e952a51a8994b5543e4ea3c1ecd/cbits/encrypted_sign.c cardano-crypto>.
+
+module Cardano.AddressDerivation.Byron
+    ( -- * Types
+      ByronKey(..)
```

Comment:
What about calling this just `Byron` :thinking: or, `Random`?

Reply:
renamed to Byron in https://github.com/input-output-hk/cardano-addresses/pull/4/commits/4034f1e2c12862632d481703f9577c1b155a479f